app/views/api_news/index.html.haml

```diff
@@ -0,0 +1,32 @@
+%div.font-thin.text-base
+    %p News
+    - @apigeneral.each do | feed |
+        %div.max-w-sm.rounded.overflow-hidden.shadow-lg
+            %h5= feed.title
```

Comment:
Does this need to be ```%h5.article-title= feed.title``` (like the other categories)? 

Also, does this map to our article attributes in some way? It isn't a problem per se here, but we will need to align these so that the editor clicking approve saves these 3rd party articles to the DB and allows them to be published. This could perhaps be addressed in the next PR though.